scripts/ci/in_container/run_ci.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+#
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+set -euo pipefail
+
+MY_DIR=$(cd "$(dirname "$0")"; pwd)
+
+if [[ ${AIRFLOW_CI_VERBOSE:="false"} == "true" ]]; then
+    set -x
+fi
+
+AIRFLOW_ROOT="${MY_DIR}/../.."
+
+PYTHON_VERSION=${PYTHON_VERSION:=3.6}
+ENV=${ENV:=docker}
+BACKEND=${BACKEND:=sqlite}
+KUBERNETES_VERSION=${KUBERNETES_VERSION:=}
+
+RUN_TESTS=${RUN_TESTS:="true"}
+
+ARGS=$@
+
+if [[ -z "${AIRFLOW_HOME:=}" ]]; then
+    echo
+    echo AIRFLOW_HOME not set !!!!
+    echo
+    exit 1
+fi
+
+# Fix file permissions
+if [[ -d $HOME/.minikube ]]; then
+    sudo chown -R airflow.airflow $HOME/.kube $HOME/.minikube
+fi
+
+if [[ ${PYTHON_VERSION} == 3* ]]; then
+    PIP=pip3
+else
+    PIP=pip2
+fi
+
+if [[ "${ENV}" == "docker" ]]; then
+    ${MY_DIR}/setup_env.sh
+    ${MY_DIR}/setup_kdc.sh
+fi
+
+if [[ "${BACKEND}" == "mysql" ]]; then
+    export AIRFLOW__CORE__SQL_ALCHEMY_CONN=mysql://root@mysql/airflow
+    export AIRFLOW__CELERY__RESULT_BACKEND=db+mysql://root@mysql/airflow
+    ${MY_DIR}/setup_mysql.sh >/dev/null
```

Comment:
Why is this done here in this image instead of using the env vars that the mysql docker image provides (similar to how we set up the postgres DB)

Reply:
Good point. I wanted to keep it close to what was done in the tox approach. I will take a look at that when I will get to fixing the failing mysql tests on Travis - maybe that's actually the reason why it fails now :).

Reply:
I changed it in latest version to use Docker-compose environment variables.